lib/awspec/helper/finder/efs.rb

```diff
@@ -0,0 +1,62 @@
+module Awspec::Helper
+  module Finder
+    module Efs
+      def find_efs(id)
+        begin
+          res = efs_client.describe_file_systems({
+                                                   file_system_id: id,
+                                                   max_items: 1
+                                                 })
+        rescue
+          # Aws::EFS::Errors::BadRequest (invalid file system ID: my-efs)
+          file_system_id = get_id_by_name_tag(id)
+          res = efs_client.describe_file_systems({
+                                                   file_system_id: file_system_id,
+                                                   max_items: 1
+                                                 })
+        end
+        res.file_systems.first if res
```

Comment:
@RyanHartje 

If res.file_systems > 1, awspec should raise `Awspec::DuplicatedResourceTypeError`.

Please use `Array#single_resource`
## Code
https://github.com/k1LoW/awspec/blob/master/lib/awspec/ext/array.rb#L2

## Usage
https://github.com/k1LoW/awspec/blob/master/lib/awspec/helper/finder/vpc.rb#L13